client/data/settings/hooks.js

```diff
@@ -10,6 +11,9 @@ import { useSelect, useDispatch } from '@wordpress/data';
  */
 import { STORE_NAME } from '../constants';
 
+/**
+ * @return {import('../../types/wcpay-data-settings-hooks').GenericSettingsHook<boolean>}
```

Comment:
May we use the `wcpay` directly to avoid path traversal? 

```suggestion
 * @return {import('wcpay/types/wcpay-data-settings-hooks').GenericSettingsHook<boolean>}
```

Reply:
For sure, I didn't think it would work on the TS docs, but it looks like it's fine! :) Thank you for the suggestion! 291e67691